src/Blef.GameLogic/Card.cs

```diff
@@ -1,12 +1,72 @@
-﻿namespace Blef.GameLogic
+using System;
+using System.Collections.Generic;
+
+namespace Blef.GameLogic
 {
-    public enum Card
+    public class Card : IComparable<Card>
```

Comment:
I see no point in comparing separate cards, what we really will compare are Poker hands.